public/index.html

```diff
@@ -25,38 +25,24 @@ <h3 id = "date"></h3>
     <br>
     
         
-        <input type="radio" value="Daily" name="dates">Every Day
-        <input type="radio" value="Monthly" name="dates">Every Month
-        <input type="radio" value="SameMonth" name="dates">Same Month Through Years:  <select name="monthsLoop">
-            <option value="0">January</option>
-            <option value="1">February</option>
-            <option value="2">March</option>
-            <option value="3">April</option>
-            <option value="4">May</option>
-            <option value="5">June</option>
-            <option value="6">July</option>
-            <option value="7">August</option>
-            <option value="8">September</option>
-            <option value="9">October</option>
-            <option value="10">November</option>
-            <option value="11">December</option>
-                                                                                      </select> 
+        <input type="radio" value="daily" name="dates">Daily
+        <input type="radio" value="monthly" name="dates">Monthly
         <br>
         
             <input type="radio" value="n" name="n-s">Arctic
             <input type="radio" value="s" name="n-s">Antarctic
             <br>
-            Starting Date (1979-2018): <input type = "date" name = "sDate">
+            <span id = "startingText"></span> <input type = "date" name = "sDate">
```

Comment:
Might be worth considering the use of a `label` tag here instead of `span`. You can tie labels directly to inputs with the `for` attribute.